src/Products/Annotation/Annotator/TextHighlightAnnotation.cs

```diff
@@ -0,0 +1,75 @@
+﻿using GroupDocs.Annotation.Models;
+using GroupDocs.Annotation.Models.AnnotationModels;
+using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+using GroupDocs.Annotation.Options;
+using System;
+using System.Collections.Generic;
+
+namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
+{
+    public class TextHighlightAnnotation : AbstractSvgAnnotator
+    {
+        private HighlightAnnotation highlightAnnotation;
+
+        public TextHighlightAnnotation(AnnotationDataEntity annotationData, PageData pageData)
+            : base(annotationData, pageData)
+        {
+            //SetFixTop(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)